TaskManager/Storages/Home/View/HomeViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  HomeViewController.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+enum Section: String, CaseIterable {
+    case Active
+    case Completed
+}
+
+protocol HomeView: AnyObject {
+    func showTask(tasks: [ModelTask])
+}
+
+class HomeViewController: UIViewController, UITableViewDelegate, HomeView {
+    
+    @IBOutlet weak private var editButton: UIButton!
+    @IBOutlet weak private var taskLabel: UILabel!
+    @IBOutlet weak private var tableView: UITableView!
+    @IBOutlet weak private var addButton: UIButton!
+    
+    private var dataSource: DataSourceDiffable!
+    var presenter: HomeViewPresenter!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    
+        configureTableView()
+        presenter.viewDidLoad()
+        addButton.setTitle("", for: .normal)
+    }
+    
+    @IBAction private func addTaskButton(_ sender: Any) {
+       
+    }
+    
+    private func configureTableView() {
+        tableView.register(UINib(nibName: String(describing: TaskTableViewCell.self), bundle: nil), forCellReuseIdentifier: TaskTableViewCell.identifier)
+        
+        dataSource = DataSourceDiffable(tableView: tableView, cellProvider: { (tableView, indexPath, itemIdentifier) -> UITableViewCell? in
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: TaskTableViewCell.identifier, for: indexPath) as? TaskTableViewCell else {return UITableViewCell()}
+            
+            switch itemIdentifier.type {
+            case .Active:
+                cell.configureCell(viewModel: itemIdentifier, backgroundColor: .white)
+                
+            case .Completed:
+                cell.configureCell(viewModel: itemIdentifier, backgroundColor: .taskManagerColor)
+                cell.strikethroughText()
+                
+            }
```

Comment:
```suggestion
            cell.configureCell(viewModel: itemIdentifier)
                
            }
```
Let's add all switch logic in configureCell, in this way you won't need to pass the backGorundColor or have aditional logic in ViewController